post-scan-actions/aws-python-promote-or-quarantine/template.yml

```diff
@@ -0,0 +1,124 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cloudone-filestorage-plugin-action-promote-or-quarantine
+    Description: >-
+      After a scan occurs, this example application places clean files in one bucket
+      and malicious files in another.
+    Author: Trend Micro Cloud One File Storage Security
+    SpdxLicenseId: Apache-2.0
+    LicenseUrl: ../../LICENSE
+    ReadmeUrl: README.md
+    Labels: [trendmicro, cloudone, filestorage, s3, bucket, plugin, promote, quarantine]
+    HomePageUrl: https://github.com/trendmicro/cloudone-filestorage-plugins
+    SemanticVersion: 1.0.0
+    SourceCodeUrl: https://github.com/trendmicro/cloudone-filestorage-plugins/tree/master/post-scan-actions/aws-python-promote-or-quarantine
+
+Parameters:
+  ScanningBucketName:
+    Type: String
+    AllowedPattern: ^[0-9a-z]+([0-9a-z-]*[0-9a-z])*$
+    ConstraintDescription: S3 bucket name can include numbers, lowercase letters,
+      and hyphens (-). It cannot start or end with a hyphen (-).
+    Description: The S3 bucket scanned by Trend Micro Cloud One File Storage Security.
+  ScanResultTopicARN:
+    Type: String
+    Description: The ARN of the scan result SNS topic in storage stack.
+  PromoteBucketName:
+    Type: String
+    Description: |
+      [Optional] The bucket name to promote files without malicious finding.
+      Leaving it blank to disable the promote capability.
+  PromoteMode:
+    Type: String
+    AllowedValues:
+      - move
+      - copy
+    Default: move
+    Description: How files got promoted.
+  QuarantineBucketName:
+    Type: String
+    Description: |
+      [Optional] The bucket name to quarantine malicious files.
+      Leaving it blank to disable the quarantine capability.
+  QuarantineMode:
+    Type: String
+    AllowedValues:
+      - move
+      - copy
+    Default: move
+    Description: How files got quarantined.
+  ACL:
+    Type: String
+    AllowedValues:
+      - ''
+      - 'private'
+      - 'public-read'
+      - 'public-read-write'
+      - 'authenticated-read'
+      - 'aws-exec-read'
+      - 'bucket-owner-read'
+      - 'bucket-owner-full-control'
+    Default: ''
+    Description: |
+      [Optional] Put ACL on the file after it's promoted/quarantined.
```

Comment:
[Optional] Apply an access control list (ACL) on the file after it has been promoted or quarantined.